project/Version.scala

```diff
@@ -1,8 +1,9 @@
 object Version {
   val awscala = "0.8.1"
   val vectorpipe = "2.0.0-SNAPSHOT"
```

Comment:
unused now since we pull version from git ?

Reply:
Unless it's valuable for local publishing for testing purposes?

Reply:
Its completely unused now:

https://github.com/geotrellis/vectorpipe/pull/120/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR8-R14

The  `gitDescribedVersion` includes a SHA for the snapshot release `git describe --tags`

Reply:
:guardsman: Aye, cap'n